packages/babel-helper-environment-visitor/src/index.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies -- TODO: Avoid cycle
```

Comment:
There are 4 packages whose types circularly depend on `@babel/traverse`.  Lets just ignore those errors for now (`@babel/traverse` will be in the dependencies anyway), but I will think of a better solution before going stable.